kotlinx-coroutines-core/common/src/flow/SharedFlow.kt

```diff
@@ -708,7 +721,21 @@ internal open class SharedFlowImpl<T>(
 
     override fun fuse(context: CoroutineContext, capacity: Int, onBufferOverflow: BufferOverflow) =
         fuseSharedFlow(context, capacity, onBufferOverflow)
-    
+
+
+    /* Entities used by IDEA debugger agent */
+
+    private class FlowValueWrapper<T>(val value: T)
+
+    private fun <T> wrap(value: T): T = value
+    private fun <T> unwrap(value: T): T = value
+
+    // debugger agent substitutes a call to this function to `wrap`
```

Comment:
This reads like
_`wrapDebuggerCapture(x)` is replaced by `wrap(x)` by the debugger agent_